### DISCUSSION TITLE
Razer Raiju wired

### DISCUSSION THREAD

**Discussion by Salocin7** [ndeadly/MissionControl#551] [Q&A] [answered]
Hi, do wired controllers work? I have the Razer Raiju and would like to know. If wired controllers don't work, is there anything I can do to be able to use it? Thank you and waiting for the answer.

**Comment by ndeadly** [comment, marked as answer]
No, wired controllers are not supported currently, only bluetooth. [sys-con](https://github.com/cathery/sys-con) exists for USB controller support, but I have no idea whether this controller works or not.

**Reply by Salocin7** [reply]
All right, i'll see sys-con. Tysm bro
